minidump/src/minidump.rs

```diff
@@ -4852,6 +4849,48 @@ mod test {
         );
     }
 
+    #[test]
+    fn test_module_list_pdb20() {
+        let name = DumpString::new("single module", Endian::Little);
+        let cv_record = Section::with_endian(Endian::Little)
+            .D32(md::CvSignature::Pdb20 as u32) // cv_signature
+            .D32(0x0) // cv_offset
```

Comment:
I have no idea what the cv_offset is supposed to contain here, but I guess it doesn't matter as that's not what I changed or am testing.